ark/utils/io_utils.py

```diff
@@ -72,23 +72,58 @@ def list_files(dir_name, substrs=None):
     return matches
 
 
-def extract_delimited_names(files, delimiter='_', delimiter_optional=True):
-    """Create a matched-index list of fov names from a list of files/folders
+def remove_file_extensions(files):
+    """Removes file extensions and drops preceding path from a list of files
 
-    Extracts a delimited prefix for every file in a given list of files
+    Args:
+        files (list):
+            List of files to remove file extensions from.
+            Any element that doesn't have an extension is left unchanged
+    Raises:
+        UserWarning:
+            Some of the processed file names still contain a period
+    Returns:
+        list:
+            List of files without file extensions
+    """
+
+    # make sure we don't try to split on a non-existent list
+    if files is None:
+        return
+
+    # only get the file name and not the directory path leading up to it
+    names = [os.path.split(name)[1] for name in files]
```

Comment:
I think we originally used this for the MIBITIFFs. Is there anywhere in the repo where we're still passing in complete paths rather than file names?

Reply:
Let me double check.